commit f39cf5251953b468e9d921e1cf2aca3abdb00772
Author: Damien Regad <dregad@mantisbt.org>
Date:   Wed Jun 24 14:52:36 2015 +0200

    Change default threshold to view project doc to VIEWER
    
    Previously it was ANYBODY, which would let any user download files from
    any project including private ones, even when they are not part of the
    team.
    
    Backport from a4be76d6e5c4939545d84712c79d3f8f4a108c4f
    
    Fixes #19873

diff --git a/config_defaults_inc.php b/config_defaults_inc.php
index 0ef20fe..374ab13 100644
--- a/config_defaults_inc.php
+++ b/config_defaults_inc.php
@@ -2354,9 +2354,13 @@
 
 	/**
 	 * Threshold needed to view project documentation
+	 * Note: setting this to ANYBODY will let any user download attachments
+	 * from private projects, regardless of their being a member of it.
+	 * @see $g_enable_project_documentation
+	 * @see $g_upload_project_file_threshold
 	 * @global int $g_view_proj_doc_threshold
 	 */
-	$g_view_proj_doc_threshold = ANYBODY;
+	$g_view_proj_doc_threshold = VIEWER;
 
 	/**
 	 * Site manager
